SU2_CFD/include/numerics_simd/flow/convection/common.hpp

```diff
@@ -79,13 +79,13 @@ FORCEINLINE void musclEdgeLimited(Int iPoint,
   auto grad_j = gatherVariables<nVar,nDim>(jPoint, gradient);
 
   for (size_t iVar = 0; iVar < nVar; ++iVar) {
-    auto proj_i = dot(grad_i[iVar], vector_ij);
-    auto proj_j = dot(grad_j[iVar], vector_ij);
-    auto delta_ij = V.j.all(iVar) - V.i.all(iVar);
-    auto delta_ij_2 = pow(delta_ij, 2) + 1e-6;
+    const Double proj_i = dot(grad_i[iVar], vector_ij);
```

Comment:
I guess you can't `const auto` here?

Reply:
On the result of "dot" I could, but not on the calculation bits, because they become an "expression" and things go wrong for reasons I do not fully understand (dangling references or whatever).